src/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ set(SRC
 	notebook.cpp
 	sqliteformat.cpp
 	sqlitestorage.cpp
+        singlesqlitebackend.cpp
```

Comment:
Here's a hard tab vs space inconsistency on the new files. Including the alarmhandler. Used to tabs but could generally prefer spaces. Either way, but consistency would be nice.

Reply:
Ah, sure sorry. I converted all to spaces to be sure that next time, Emacs adding spaces by default will not mix up things with existing TABs.